File-Formats/DocIO/Working-with-LaTeX.md

```diff
@@ -1,12 +1,12 @@
 ---
 title: Working with LaTeX | DocIO | Syncfusion
-description: Learn how to create mathematical equations in a Word document using LaTeX with the .NET Word (DocIO) library without Microsoft Word.
+description: Learn all about how to create mathematical equations in Word documents using LaTeX with Syncfusion’s essential .NET Word library for Xamarin.
```

Comment:
why it was removed?
<img width="662" alt="image" src="https://github.com/syncfusion-content/fileformat-docs/assets/61148953/e233329c-ebe8-45e4-af62-adf0e853fce4">

Reply:
resolved